xauc/probability_tests.py

```diff
@@ -0,0 +1,78 @@
+# Lint as: python3
+"""Tests the probability functions in probability.py.
+
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import unittest
+import numpy as np
+import probability
+
+class ProbabilityFunctionsTest(unittest.TestCase):
+    """Unit tests for functions in probability.py.
+
+    Attrs:
+        maximum_error: upper bound on difference between expected and calculated
+            value.
+    """
+
+    maximum_error = 1e-6
+
+    def test_probability_functions_unordered_different_sizes(self):
```

Comment:
What value is returned:
-  if scores_a or scores_b is empty?
-  if all scores are the same
- And perhaps one sanity check is to test the case in which scores_a are strictly higher than scores_b to verify prob = 1/0

Reply:
I'll add those tests!